src/runtime/Object.cpp

```diff
@@ -861,21 +863,21 @@ bool Object::set(ExecutionState& state, const ObjectPropertyName& propertyName,
     return true;
 }
 
-bool Object::ordinaryHasProperty(ExecutionState& state, Object* object, const ObjectPropertyName& propertyName)
+bool Object::ordinaryHasProperty(ExecutionState& state, const ObjectPropertyName& propertyName)
 {
```

Comment:
According to the [spec](https://www.ecma-international.org/ecma-262/6.0/#sec-ordinaryhasproperty), `Object::ordinaryHasProperty` method has two parameters (`OrdinaryHasProperty (O, P)`) which are Object O and PropertyKey P. To match the spec, object parameter seems to be maintained.

Reply:
Indeed, but the `O` always represents the object itself to check whether it has property or not. So at this point the `O` can be substituted with the `this` argument.